onnxruntime/core/providers/tensorrt/tensorrt_execution_provider.h

```diff
@@ -146,6 +149,12 @@ class TensorrtExecutionProvider : public IExecutionProvider {
 
   void RegisterStreamHandlers(IStreamCommandHandleRegistry& stream_handle_registry) const override;
 
+  // Helper to generate engine id via model name/model content/env metadata
+  static int TRTGenerateId(const GraphViewer& graph_viewer, HashValue& model_hash);
+
+  // Call TRTGenerateModelId to generate hash id for TRT engine cache
+  static int TRTGenerateModelId(const GraphViewer& graph_viewer, HashValue& model_hash);
```

Comment:
Why do you make those two functions static?

Reply:
Those two functions are removed from trtep.h in latest commit, as I am planning to keep cache-related functions in utility.h and not moving to class TRTEP

Reply:
Since now they are class functions, not utility functions anymore, they should not be in the utility file.